tests/foreman/api/test_docker.py

```diff
@@ -1295,6 +1296,7 @@ def test_positive_delete(self):
                     container.read()
 
 
+@run_in_one_thread
```

Comment:
URL for the registry should be unique and we have only 1 such available. So we should run registry tests only in 1 thread to wait each time until registry gets deleted.